packages/mdc-typography/_mixins.scss

```diff
@@ -47,3 +51,11 @@
   white-space: nowrap;
   overflow: hidden;
 }
+
+@mixin mdc-typography-button {
```

Comment:
I think this mixin should be in MDC Button package, and maybe renamed to `mdc-button-text`

And you should replace line 46(ish) in mdc-button.scss with `@include mdc-button-text`

Does `user-select: none;` and `-webkit-appearance: none` really belong in this mixin?